components/Boards/PlayerVsPlayer.js

```diff
@@ -0,0 +1,112 @@
+import { Chessboard } from "react-chessboard";
+import Chess from "chess.js";
+import { useState, useEffect } from "react";
+import MoveInfo from "../MoveInfo";
+import PlayerNameDisplay from "../PlayerNameDisplay";
+import GameTerminal from "../GameTerminal";
+
+export default function RandomMoveEngine() {
+  const [game, setGame] = useState(null);
+  const [fen, setFen] = useState(game?.fen());
```

Comment:
does this need to be its own state or can it be a derived state from your game state? 🤔

Reply:
This was one of the things that went out of sync with the game state over time, so I gave it its own state and placed the setter inside the zombie-function and inside makeAMove. The chessboard `position={fen}` also updates correctly now, with fen state (the fen state only gets updated, when I want it to, and not automatically together with the game state). Im not exactly sure there is another solution, but this one works.

I will also make use of the fen state in the future, when I need to collect all fens of a game for the database.

Reply:
Alright, then it makes sense to have a separate state for this 👍